openquakeplatform/openquakeplatform/ript/static/ript/js/vulnerability_new.js

```diff
@@ -0,0 +1,525 @@
+                '<input id="'+vCount+'" class="id vfsTable" type="text"><br>' +
+                '<label> IMT: </label>' +
+                '<select id="imtSelection'+vCount+'" class="imtSelection vfsTable">'+
+                    '<option value="pga">PGA (g)</option>'+
+                    '<option value="pgv">PGV (cm/sec)</option>'+
+                    '<option value="pgd">PGD (cm)</option>'+
+                    '<option value="sa">SA (T)</option>'+
+                    '<option value="ia">IA (m/s)</option>'+
+                    '<option value="cav">CAV</option>'+
+                    '<option value="rsd">RSD</option>'+
+                    '<option value="mmi">MMI</option>'+
+                '</select>'+
+                '<div id="saValueInputDiv'+vCount+'">'+
+                    '<button id="saValue'+vCount+'" type="button" class="btn btn-default" aria-label="Left Align">' +
+                        '<span class="glyphicon glyphicon-plus" aria-hidden="true">+</span>' +
+                    '</button>'+
+                    '<input id="saValueInput'+vCount+'" class="saValue vfsTable" type="text" placeholder="0.05"><br>' +
+                '</div>'+
+                '<label> Dist: </label>' +
+                '<select id="dist'+vCount+'" class="vfsTable distributionSelect">' +
+                    '<option value="BT">BT</option>' +
+                    '<option value="LN">LN</option>' +
+                    '<option value="PM">PM</option>' +
+                '</select>'+
+                '<div id="imldiv'+vCount+'">'+
+                    '<label>IML: </label>' +
+                    '<input id="imlArray'+vCount+'" class="imls vfsTable" placeholder="tab or comma separated values" type="text">' +
+                    '<button id="imlArrayInput'+vCount+'" type="button" class="imlBtn btn btn-default" aria-label="Left Align">' +
+                        '<span class="glyphicon glyphicon-plus" aria-hidden="true">+</span>' +
+                    '</button>'+
+                '</div>' +
+            '</div>'+
+            '<div id="tableOverflow">'+
+                '<div id="vTableDiv'+vCount+'" class="theVTable"></div><br><br>' +
+                '<br><br>' +
+                '</div>' +
+            '</div>' +
+        '</div>'
+    );
+
+    $('#saValueInputDiv'+vCount).hide();
+    $('#imldiv'+vCount).hide();
+
+    // force bootstrap style
+    $('.btn-danger').css({'background-color': '#da4f49'});
+}
+
+function createVulnDiscreteTable(vFormat, selectedImlPrint, vCount, imlArray) {
+
+    // Argument '4' is the SA values (when provided)
+    if (arguments[4]) {
+        selectedImlPrint = 'SA ('+arguments[4]+')';
+    }
+
+    var colWidth, maxRows;
+
+    // Setup the header
+    if (vFormat == 'probability mass function') {
+        // Add selectedImlPrint to the begining of an imlArrayInput
+        imlArray.unshift(selectedImlPrint);
+
+        vHeader = imlArray;
+        //vHeader = ['loss ratio', 'probabilities'];
```

Comment:
For the vulnerability_new.js file I prefer to keep it as reference as it is, I add a comment to explain.